benchmarking/src/performance_evaluation.py

```diff
@@ -324,10 +358,42 @@ def create_output_filename(self) -> str:
         generation_mode = ""
         if self.is_stream_mode:
             generation_mode = "stream"
-    
-        return f"{self.model_name}_{self.file_name}_{self.num_workers}_{generation_mode}"
+        
+        output_file_name = f"{self.model_name}_{self.file_name}_{self.num_workers}_{generation_mode}"
+        return self.sanitize_file_prefix(output_file_name)
+
+    def save_results(
+        self,
+        filename: str,
+        summary: Dict[str, Any], 
+        individual_responses: List[LLMResponse]
+    ):
```

Comment:
nit: We should add a docstring for this.

Reply:
dostring added, thanks!